lib/index.js

```diff
@@ -326,13 +373,14 @@ exports.deleteStorage = async (ctx, storageCtx) => {
     debug('Deleted issuer');
 };
 
-exports.createFunction = async (ctx, functionSpecification) => {
+exports.createFunction = async (ctx, functionSpecification, accessToken) => {
```

Comment:
It could. We can add it later as needed in a non-breaking way. The difference between createStorage and createFunction is that createStroage has enough context to determine the intention of the caller (absence of the PKI in configuration), while createFunction does not.

Reply:
Removed createStorage